.github/CONTRIBUTING.md

```diff
@@ -1,64 +1,65 @@
-2. Try to prevent any breaking changes affecting the end-user (Developers using the API).  
-Mention any breaking changes that could affect the end-user.
-3. Document any public methods that the end-user might use.
+### Follow the Template
+We have Issue templates to help us get the required information more easly. Please follow the provided template when either filing a bug report or feature request.  
+We reserve the right to close your issue without warning for not following the templates.
 
-## Code of Conduct
-
-### Our Pledge
-
-In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, nationality, personal appearance, race, religion, or sexual identity and orientation.
-
-### Our Standards
+### Using Latest Version
+When it comes to bug reports should you always check first, if you're using the latest release of PlaceholderAPI.  
+Often is a bug you encounter fixed in a newer release.
 
-Examples of behavior that contributes to creating a positive environment
-include:
+You can optionally also try out development builds to see, if your bug is fixed in those.
 
-* Using welcoming and inclusive language
-* Being respectful of differing viewpoints and experiences
-* Gracefully accepting constructive criticism
-* Focusing on what is best for the community
-* Showing empathy towards other community members
+The same rules apply for when making a feature request.
 
-Examples of unacceptable behavior by participants include:
+### No duplicate issue
+Make sure that there isn't any existing issue, which is still open, or is closed with a solution/explanation.  
+Opening a separate issue for a bug report or feature request, that already exists on the issue tracker only slows down the process of fixing the bug or implementing the feature.
 
-* The use of sexualized language or imagery and unwelcome sexual attention or advances
-* Trolling, insulting/derogatory comments, and personal or political attacks
-* Public or private harassment
-* Publishing others' private information, such as a physical or electronic address, without explicit permission
-* Other conduct which could reasonably be considered inappropriate in a professional setting
+If an issue with the bug or feature you want to report/suggest exists, comment on it with your info (bug reports) or give it a :thumbsup: (Feature Request) to show that this is important for you.
 
-### Our Responsibilities
+### Issue isn't caused by external source
+PlaceholderAPI provides a feature to have extensions (separate jar files) for placeholders. This gives it a possability that an issue you encounter is caused by said extensions or a separate plugin that uses those extensions.  
```

Comment:
Yeah, they're extensions, but be consistent with your naming. Call them expansions, not extensions.